chameleon/ChameleonOIDCAuthBackend.py

```diff
@@ -0,0 +1,37 @@
+from mozilla_django_oidc.auth import OIDCAuthenticationBackend
+from chameleon.models import UserProperties
+import logging
+
+logger = logging.getLogger('auth')
+
+class ChameleonOIDCAB(OIDCAuthenticationBackend):
+    def create_user(self, claims):
+        if self.get_settings('OIDC_USERNAME_ALGO', None):
```

Comment:
I don't quite get what this branch is doing - why do we check this particular setting and skip the rest of the flow here?